cscore/examples/usbviewer/usbviewer.cpp

```diff
@@ -20,10 +24,10 @@
 namespace gui = wpi::gui;
 
 int main() {
-  std::atomic<cv::Mat*> latestFrame{nullptr};
-  std::vector<cv::Mat*> sharedFreeList;
-  wpi::spinlock sharedFreeListMutex;
-  std::vector<cv::Mat*> sourceFreeList;
+  wpi::mutex latestFrameMutex;
+  std::unique_ptr<cv::Mat> latestFrame;
+  wpi::spinlock freeListMutex;
```

Comment:
```suggestion
  wpi::spinlock latestFrameMutex;
  std::unique_ptr<cv::Mat> latestFrame;
  wpi::mutex freeListMutex;
```
The latest frame mutex can be a spinlock, as the only operation it's protecting is a swap operation.  The free list mutex should be a normal mutex, because there's some long running things that can happen while it's held (e.g. allocating memory).